modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -60,3 +60,7 @@
 	right: 0;
 	top: 0;
 }
+
+.c-empty-state-animation {
```

Comment:
hey @ilzamcmed could you please sort this? Thanks!

Reply:
Fixed! Thanks @fmarcio!